common-rendering/src/editorialPalette/background.ts

```diff
@@ -125,13 +124,51 @@ const standfirstDark = ({ design, theme }: ArticleFormat): Colour => {
 	}
 };
 
+const liveblogStandfirstBulletDark = ({
+	design,
+	theme,
+}: ArticleFormat): Colour => {
+	switch (design) {
+		case ArticleDesign.DeadBlog:
+			return neutral[46];
+		case ArticleDesign.LiveBlog:
+			switch (theme) {
+				case ArticlePillar.Opinion:
+					return opinion[550];
+				case ArticlePillar.Sport:
+					return sport[500];
+				case ArticlePillar.Culture:
+					return culture[500];
+				case ArticlePillar.Lifestyle:
+					return lifestyle[500];
+				case ArticlePillar.News:
+					return news[550];
+				default:
+					return neutral[46];
```

Comment:
You may want the other themes (labs and special reports) to use the same colour as news. See https://github.com/guardian/dotcom-rendering/pull/4195#discussion_r820642373

Reply:
That's an odd one, I see your point and I agree but I think I just copied the switch from the standfirst background so is there a wider issue of missing styles for those special article types.